timify-api/app.js

```diff
@@ -29,8 +30,13 @@ async function taddyGraphqlRequest(query, variables) {
       'X-API-KEY': `${process.env.API_KEY}`,
     }
   });
+  console.log(response.data)
   return response.data;
 }
```

Comment:
I think this bracket is also incorrectly formatted? looks like it belongs to the try catch statement but it's not aligned.